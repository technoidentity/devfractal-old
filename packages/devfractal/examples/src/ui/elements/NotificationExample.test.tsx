import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Notification } from './NotificationExample'

test('Notification Example', () => {
  const component: ReactTestRenderer = renderer.create(<Notification />)
  expect(component.toJSON()).toMatchSnapshot()
})
