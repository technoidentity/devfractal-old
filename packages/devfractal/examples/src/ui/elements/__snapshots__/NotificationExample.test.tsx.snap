// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Default Notification
    </h1>
    <div
      className="box"
    >
      <div
        className="notification"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Colors
    </h1>
    <div
      className="box"
    >
      <div
        className="notification is-primary"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-success"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-warning"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-danger"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-white"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-black"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-dark"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
      <div
        className="notification is-light"
      >
        Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something Bold notification blocks, to alert your users of something
        <a
          className="delete"
        />
      </div>
    </div>
  </section>,
]
`;
